jmxtrans-output/jmxtrans-output-logback/src/main/java/com/googlecode/jmxtrans/model/output/LogbackWriter.java

```diff
@@ -88,7 +90,7 @@ public void internalWrite(Server server, final Query query, ImmutableList<Result
 
 				MDC.put("server", alias);
 				MDC.put("metric", KeyUtils.getKeyString(server, query, result, typeNames, null));
-				MDC.put("value", result.getValue());
+				MDC.put("value", result.getValue().toString());
```

Comment:
All those MDC entries should be cleaned up after the call to log.info()

Reply:
I fixed it, but the bug was already in Log4J1 edition.
IMHO we should ditch the outputwriter in favor of jmxtrans-output/jmxtrans-output-core/src/main/java/com/googlecode/jmxtrans/model/output/Slf4JOutputWriter.java

Reply:
Yep, I'm all for ditching it!